versioned/tiered/gc/src/test/java/org/projectnessie/versioned/tiered/gc/ITTestIdentifyUnreferencedAssets.java

```diff
@@ -129,9 +122,14 @@ public void run() throws Exception {
         .maxAgeMicros(ONE_DAY_OLD_MICROS)
         .timeSlopMicros(ONE_HOUR_OLD_MICROS)
         .build();
-    IdentifyUnreferencedAssets<DummyValue> app = new IdentifyUnreferencedAssets<DummyValue>(helper, new DynamoSupplier(), spark, options);
-    Dataset<UnreferencedItem> items = app.identify();
-    Set<String> unreferencedItems = items.collectAsList().stream().map(UnreferencedItem::getName).collect(Collectors.toSet());
+    IdentifyUnreferencedValues<DummyValue> app = new IdentifyUnreferencedValues<>(helper, new DynamoSupplier(), spark, options);
```

Comment:
I think it probably makes sense to decompose the tests into tests for Assets and Values separately.

Reply:
I agree. I have added extra asserts to test the values. I wasn't sure how to split it up into separate tests as the 2nd half is so reliant on the first. 

I did add a new test to test the Asset search module which is independent of the value search